### ISSUE TITLE
iOS 14 handling preferredDatePickerStyle

### ISSUE THREAD

**Issue by nicholaschong95** [squimer/DatePickerDialog-iOS-Swift#111]
Hi, DatePickerDialog will be having issue especially in iOS 14, as preferredDatePickerStyle by default is in compact mode, new picker style will be inline, any idea to fix ?

**Comment by gmanenti** [comment]
Hi, just insert:

`if #available(iOS 13.4, *) {
            datePicker.preferredDatePickerStyle = .wheels
     }`

in the show function.

**Comment by lexin** [comment]
yes. It would be nice to prepare pod update with this line just in the lib
